src/assignment/helpers/delete-assignment.ts

```diff
@@ -48,6 +51,20 @@ export async function deleteAssignment(
 }
 
 export function deleteAssignmentWarning(assignment?: Avo.Assignment.Assignment): ReactNode {
+	const isSharedWithOthers =
+		assignment?.share_type === ShareWithColleagueTypeEnum.GEDEELD_MET_ANDERE;
+	const isContributor = assignment?.share_type === ShareWithColleagueTypeEnum.GEDEELD_MET_MIJ;
+
+	if (isSharedWithOthers) {
+		return tHtml('assignment/views/assignment-overview___delete-shared-assignment', {
+			count: assignment?.contributors?.length,
```

Comment:
if you use optional chaining, you must provide a default. Otherwise it might write out "undefined" in the string
```suggestion
			count: assignment?.contributors?.length || 0,
```